packages/grid/test/selection.test.js

```diff
@@ -509,4 +510,52 @@ describe('multi selection column', () => {
 
     expect(grid.selectedItems).to.eql(grid.items);
   });
+
+  it('should select items on mouse drag when selectRowsByDragging is enabled', () => {
+    grid.items = ['foo', 'bar', 'baz', 'qux', 'quux', 'corge'];
+    flushGrid(grid);
+    rows = getRows(grid.$.items);
+
+    selectionColumn.selectRowsByDragging = true;
+
+    const sourceCell = getCellContent(getRowCells(rows[0])[0]);
```

Comment:
Test might be more useful if it doesn't start at the first row since part of the tested logic is to figure out in which row dragging started.

Reply:
Done